src/metabase/util/performance.clj

```diff
@@ -126,3 +126,8 @@
       ([x y] (mapv #(% x y) fns))
       ([x y z] (mapv #(% x y z) fns))
       ([x y z & args] (mapv #(apply % x y z args) fns)))))
+
+(defn some
```

Comment:
Is this worth using in place of `clojure.core/some` in all cases, or only in performance-sensitive places?

Reply:
It would be a drop-in replacement, but I don't see much value in going over the whole codebase and polluting `git blame` :). Maybe, at some point, I will come up with a more generic drop-in core lib replacement. Until then, it makes sense to do a pinpoint replacements where it is necessary.